test/tracerEq/test_consistency_2d.py

```diff
@@ -85,42 +90,55 @@ def run_tracer_consistency(constant_c = True, **model_options):
     if options.solve_tracer:
         tracer_int, tracer_int_rerr = solver_obj.callbacks['export']['tracer_2d mass']()
         assert abs(tracer_int_rerr) < 1e-4, 'tracer is not conserved'
+        tracer_mass_int, tracer_mass_int_rerr = solver_obj.callbacks['timestep']['tracer_2d total mass']()    
+        print_output("Tracer total mass error: %11.4e" %(tracer_mass_int_rerr))
+        assert abs(tracer_mass_int_rerr) < 1.2e-4, 'total tracer mass is not conserved'
         smin, smax, undershoot, overshoot = solver_obj.callbacks['export']['tracer_2d overshoot']()
         max_abs_overshoot = max(abs(undershoot), abs(overshoot))
         overshoot_tol = 1e-12
-        msg = 'Tracer overshoots are too large: {:}'.format(max_abs_overshoot)
-        assert max_abs_overshoot < overshoot_tol, msg
-
+        if options.use_tracer_conservative_form == False:
```

Comment:
Don't use `<logical> == False`, say `not <logical>` instead.

Reply:
Changed thanks for the comment!